tests/core/test_farmer_harvester_rpc.py

```diff
@@ -169,37 +170,66 @@ async def have_plots():
             res_2 = await client_2.get_plots()
             assert len(res_2["plots"]) == num_plots
 
+            # Reset cache and force updates cache every second to make sure the farmer gets the most recent data
+            update_interval_before = farmer_api.farmer.update_harvester_cache_interval
+            farmer_api.farmer.update_harvester_cache_interval = 2
```

Comment:
I'm not convinced the increased complexity of sneaking in this parameter as a member here, to affect the behavior of this class later just for testing purposes carries its weight.

If it should be configurable, it should probably be a proper configuration option.

Reply:
I didn't put it into the config because it will be dropped soon anyway because i'm going to change the behavior so that the harvester reports the plots instead of the farmer requests them.